test/functional/p2p_segwit.py

```diff
@@ -148,7 +148,7 @@ def test_witness_block(node, p2p, block, accepted, with_witness=True, reason=Non
 
 class TestP2PConn(P2PInterface):
     def __init__(self, wtxidrelay=False):
-        super().__init__()
+        super().__init__(wtxidrelay=wtxidrelay)
```

Comment:
Can you make the changes to the test framework in a separate commit?

Reply:
Sure, I did it at first before to squash. Split in two.